package.json

```diff
@@ -33,21 +33,30 @@
     "nexus": "^0.12.0-rc"
   },
   "devDependencies": {
-    "@prisma/client": "^2.0.0-preview020",
+    "@prisma/cli": "^0.1.31",
+    "@prisma/client": "^2.0.0-preview020.3",
     "@prisma/fetch-engine": "^0.3.52",
+    "@prisma/lift": "^0.3.384",
+    "@prisma/sdk": "^0.0.176",
     "@types/fs-extra": "^8.0.0",
     "@types/jest": "^24.9.0",
     "@types/node": "^13.1.8",
     "@types/pluralize": "^0.0.29",
+    "@types/rimraf": "^2.0.3",
     "@types/tmp": "^0.1.0",
     "doctoc": "^1.4.0",
     "dripip": "^0.5.0",
+    "get-port": "^5.1.1",
     "graphql": "^14.5.4",
+    "graphql-request": "^1.8.2",
+    "graphql-yoga": "^1.18.3",
     "jest": "^24.9.0",
     "jest-watch-typeahead": "^0.4.0",
     "nexus": "^0.12.0-rc.9",
     "prettier": "^1.17.1",
-    "prisma2": "^2.0.0-preview020",
+    "prisma2": "^2.0.0-preview020.3",
+    "react": "^16.12.0",
```

Comment:
Reason why we need `react`, and `@prisma/cli` is because of `@prisma/lift` which has them as peerDep